cmd/state/forward.go

```diff
@@ -36,7 +36,7 @@ func forwardFn(bindir string, args []string, out output.Outputer, pj *project.Pr
 	}
 
 	// Retrieve the version info specified in the activestate.yaml
-	versionInfo, err := projectfile.ParseVersionInfo(pj.Source().Path())
+	versionInfo, err := project.ParseVersionInfo(pj.Source().Path())
```

Comment:
This one doesn't make sense. It takes the file of an activestate.yaml as the source, which is decidedly a `projectfile` responsibility.